### PR TITLE
add test new duplicate name project

### PR DESCRIPTION
**Pull request by ke-de-ve**

### CODE REVIEW

File: src/test/java/school/redrover/NewItemTest.java [hunk 1]
```diff
@@ -249,4 +250,28 @@ public void testItemNameWithUnsafeSpecialCharacterNotAllowed() {
...
+    }
+
+    @Test(dependsOnMethods = "testCreateNewItemWithCorrectName")
+    public void testCreateNewItemWithExistingName() {
+        String errorMessage = new HomePage(getDriver())
+                .clickNewItemOnLeftSidePanel()
+                .sendItemName(NEW_ITEM_NAME + "\t")
+                .getItemNameInvalidMessage();
+
+        Assert.assertEquals(errorMessage, "» A job already exists with the name ‘%s’".formatted(NEW_ITEM_NAME));
```
Comment (romasky) [inline src/test/java/school/redrover/NewItemTest.java:275]: А если текст будет на другом языке? Может быть проверять что появляется само сообщение?
